pydarn/io/superdarn_formats.py

```diff
@@ -127,6 +125,10 @@ class Fitacf():
         'ptab': 'h',
         'ltab': 'h',
         'pwr0': 'f'}
+    extra_fields = {
```

Comment:
Would you mind adding comments above/at each of these variables that say what they are and specify the units they use?

Reply:
Actually to save time and keep consistency with newly updated changes, I will add a reference (s) to where these variables are described in RST documentation.